docs/BSD.md

```diff
@@ -0,0 +1,84 @@
+# BSD building/testing instructions
+
+BSD is not officially supported.
+These are some basic instructions for getting CFEngine Community to build on FreeBSD.
+Tested on FreeBSD 11 VM in AWS EC2.
+
+## Dependencies
+
+```
+$ su root -c 'pkg update -f'
+$ su root -c 'pkg install -y gdb gcc gmake lmdb autoconf automake libtool git python3'
```

Comment:
to make this "out of the box" maybe change all "su root -c" to "doas <cmd>". su isn't installed by default and you don't include that step here as a dep right?"

Reply:
su is installed by default on the FreeBSD 11 VM Image in AWS. Guide says that's where it's tested. I used su, since that's something people are familiar with from other unixes / linuxes.

Reply:
Cool, yeah I'm no FreeBSD expert but I'm pretty sure "doas" is more the default. Will check it out. :+1: